src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -544,13 +774,14 @@ public String processZip(UploadedFile file) {
         }
         return jsonStrSessionStorage;
     }
-
+*/
     /**
      * Accepts a POST request to to initiate a quick save (download a temp save
      * template)
      *
      * @return A String of the local file name for the ASCII file (or null for an error).
      */
+    /*
     @RequestMapping(value = "/QuickSave", method = RequestMethod.POST)
     @ResponseBody
     public String quickSave(String jsonStrSessionStorage) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)